tests/delete_regress/models.py

```diff
@@ -151,3 +148,38 @@ class OrderedPerson(models.Model):
 
     class Meta:
         ordering = ["name"]
+
+
+# Models for #35073
```

Comment:
I know that you follow a pattern of previous test but we no longer add such comments.
```suggestion
```

Reply:
fixed